include/dxc/WinAdapter.h

```diff
@@ -934,26 +933,16 @@ class ScopedLocale {
       assert(false && "Support for Linux only handles UTF8 code pages");
     }
 #ifdef __APPLE__
-    switch (CodePage) {
-    case CP_UTF8:
-      return setlocale(Category, "en_US.UTF-8");
-    default:
-      return nullptr;
-    }
+    return setlocale(Category, "en_US.UTF-8");
 #else
     const char *utf8LocaleOptions[] = {"en_US.UTF-8", "en_US.utf8"};
```

Comment:
I don't think we need `en_US.utf8` anywhere.  `en_US.UTF-8` looks like it is pretty universal, it's just an outlier that Ubuntu also supports the lowercase version.